doc/sphinxext/numpydoc/numpydoc.py

```diff
@@ -43,7 +43,10 @@ def mangle_docstrings(app, what, name, obj, options, lines,
               )
 
     # PANDAS HACK (to remove the list of methods/attributes for Categorical)
-    if what == "class" and name.endswith(".Categorical"):
+    if what == "class" and (name.endswith(".Categorical") or
+                            name.endswith("CategoricalIndex") or
+                            name.endswith("Interval") or
```

Comment:
@TomAugspurger I think Interval class itself is OK to have with all its methods / attributes ?